presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -545,6 +546,26 @@ public void testDropTableIfExists()
     }
 
     @Test
+    public void testOrderByChar()
+            throws Exception
+    {
+        assertUpdate("CREATE TABLE char_order_by (c_char char(2))");
```

Comment:
not all connectors support char (e.g raptor). Maybe remote this test or move it to: `TestHiveDistributedQueries`?